.github/workflows/ci-backend-cql.yml

```diff
@@ -98,18 +98,18 @@ jobs:
             java: 8
           - module: cql
             install-args: "-Pjava-11"
```

Comment:
This file contains 12 test entries for Java 8, but only 3 for Java 11. Shouldn't all tests be also executed with Java 11?